patroni/patroni.spec

```
@@ -42,6 +42,8 @@ Requires:      /usr/bin/python3.6, libffi, postgresql-server, libyaml, postgresq
 BuildRequires: prelink libyaml-devel gcc
 Provides:      patroni
 Epoch:         1
+Packager:       Percona Development Team <https://jira.percona.com>
+Vendor:         Percona, Inc
```

Note that this should read `Percona, LLC` (see [DISTPG-341](https://jira.percona.com/browse/DISTPG-341)):
```suggestion
Vendor:         Percona, LLC
```

---

done